src/iop/demosaic.c

```diff
@@ -1641,17 +1640,17 @@ void process(struct dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *piece, const
         switch(data->green_eq)
         {
           case DT_IOP_GREEN_EQ_FULL:
-            green_equilibration_favg(in, pixels, roi_in->width, roi_in->height, piece->pipe->filters, roi_in->x,
-                                     roi_in->y);
+            green_equilibration_favg(in, pixels, roi_in->width, roi_in->height, piece->pipe->dsc.filters,
+                                     roi_in->x, roi_in->y);
             break;
           case DT_IOP_GREEN_EQ_LOCAL:
-            green_equilibration_lavg(in, pixels, roi_in->width, roi_in->height, piece->pipe->filters, roi_in->x,
-                                     roi_in->y, 0, threshold);
+            green_equilibration_lavg(in, pixels, roi_in->width, roi_in->height, piece->pipe->dsc.filters,
+                                     roi_in->x, roi_in->y, 0, threshold);
             break;
           case DT_IOP_GREEN_EQ_BOTH:
-            green_equilibration_favg(in, pixels, roi_in->width, roi_in->height, piece->pipe->filters, roi_in->x,
-                                     roi_in->y);
-            green_equilibration_lavg(in, in, roi_in->width, roi_in->height, piece->pipe->filters, roi_in->x,
+            green_equilibration_favg(in, pixels, roi_in->width, roi_in->height, piece->pipe->dsc.filters,
```

Comment:
this is just a line wrapping change.. but it reminds me. is the filters variable on dsc now independent on roi->{x,y}? or do we still need to offset it according to the region of interest?